src/main/java/org/springframework/data/couchbase/core/query/QueryCriteriaDefinition.java

```diff
@@ -18,8 +18,11 @@
 /**
  * @author Oliver Gierke
  * @author Christoph Strobl
+ * @author Michael Reiche
  */
 public interface QueryCriteriaDefinition {
 
+	String export(int[] paramIndexPtr);
```

Comment:
Public API should be documented properly with Javadoc.

Reply:
fixed. just now, this wasn't in the earlier --amend

Reply:
is this already addressed?